vermouth/tests/test_annotate_mut_mod.py

```diff
@@ -17,6 +17,8 @@
 
 import networkx as nx
 import pytest
+from contextlib import nullcontext as does_not_raise
```

Comment:
This is not actually used, right?

Reply:
yep, will remove that and next line, used when trying to work out how to sort out the tests.